src/registrar/admin.py

```diff
@@ -706,17 +757,33 @@ class HostIPInline(admin.StackedInline):
     model = models.HostIP
 
 
-class MyHostAdmin(AuditedAdmin):
+class HostResource(resources.ModelResource):
+
```

Comment:
**suggestion/non-blocking** We do typically have it as a rule that every class needs at least a small comment on its purpose. I would suggesting writing a single line describing the purpose (nothing too complicated) and basically making it generic so that you can paste it below the other "Resource" classes and call it good. Suggesting something like the below
```suggestion
defines how each field in the referenced model should be mapped to the corresponding fields in the import/export file
```